backend/src/auth_service/opaque.rs

```diff
@@ -44,6 +44,7 @@ mod persistence {
             &self,
             connection: impl PgExecutor<'_>,
         ) -> Result<(), StorageError> {
+            // TODO: Should we make sure that there is only one opaque setup?
```

Comment:
We probably should. Similar to the AsCredentials, we don't really have much in the way of public key management at the moment. I'd leave it for now and worry about it when we add said management functionalities.